src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -1121,6 +1125,15 @@
       <SupportedFramework Condition="'%(SupportedFramework.Version)' == ''">
         <Version>$(_AssemblyVersion)</Version>
       </SupportedFramework>
+
+      <!--
+        Harvesting supported versions can sometimes lead to unknown versions when harvesting a package
+        which has placeholders for older frameworks like net45. We should just exclude those from validation
+        since we cannot easily determine what they are without looking at each specific platform itself.
+      -->
+      <ItemGroup>
+        <SupportedFramework Remove="@(SupportedFramework)" Condition="'%(SupportedFramework.Version)' == 'unknown'" />
```

Comment:
I didn't look but I assume that you updated your corefx PR to remove this from there since it will live here now.

Reply:
I didn't update my PR yet. Both of these will work with them being in both places. I will either update my PR or remove them from corefx once this gets merged whichever happens first.